src/Components/Web/src/Forms/InputSelect.cs

```diff
@@ -10,14 +11,14 @@ namespace Microsoft.AspNetCore.Components.Forms
     /// <summary>
     /// A dropdown selection component.
     /// </summary>
-    public class InputSelect<TValue> : InputBase<TValue>
+    public class InputSelect<TValue> : InputBase<TValue> where TValue : notnull
```

Comment:
@SteveSandersonMS, this might play a little better with https://github.com/dotnet/aspnetcore/pull/23221.

Reply:
Not sure this is right. Are you killing the option to bind to a nullable enum? We don't want to drop support for that (unless I'm misunderstanding and somehow we don't currently support it).

Reply:
I can roll this back. I guess I misunderstood what you were implying in the other PR.